pipeline/deploy.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set +x
+
+if [[ -z "${VSPHERE_CSI_CONTROLLER_IMAGE}" ]]
+then
+	echo "Env variable unset: VSPHERE_CSI_CONTROLLER_IMAGE"
+	exit 1
+fi
+
+if [[ -z "${VSPHERE_SYNCER_IMAGE}" ]]
+then
+	echo "Env variable unset: VSPHERE_SYNCER_IMAGE"
+	exit 1
+fi
+
+# Borrow a testbed from CSI Testbed Pool Svc.
+testbed=`curl -X 'GET' ${CNS_TESTBEDPOOL_SVC_URL}`
+if [ $? -ne 0 ]
+then
+	echo "Unable to borrow a testbed!"
+	exit 1
+fi
+echo "TestbedInfo: $testbed"
+
+vcIp=`echo $testbed | jq '.vcIp'|tr -d '"'`
+if [ $? -ne 0 ]
+then
+	echo "Error getting the vcIp!"
+	exit 1
+fi
+
+echo "VC_IP=$vcIp" >> build.env
+
+svAdminCreds=`echo $testbed | jq '.svAdminCreds'|tr -d '"'|base64 -d`
+if [ $? -ne 0 ]
+then
+	echo "Error getting the svAdminCreds!"
+	exit 1
+fi
+echo "svAdminCreds = $svAdminCreds"
+SV_KUBECONFIG=/tmp/$$
+echo $testbed | jq '.svAdminCreds'|tr -d '"'|base64 -d > $SV_KUBECONFIG
+export KUBECONFIG=$SV_KUBECONFIG
+
+kustomize build pipeline/dev | envsubst | kubectl apply -f -
+if [ $? -ne 0 ]
+then
+	echo "Error patching the CSI images in the testbed!"
+	exit 1
+fi
+
+# Sleep for 60 seconds so that k8s can act on the applied changes.
+echo "Sleeping for 60 seconds..."
```

Comment:
Do we have anything in `kustomize build` to wait until patches are applied, so we do not need to wait for random 60 seconds?

Reply:
I'm not aware of any option in `kustomize` that waits until the applied patches are successfully processed by k8s. Let me know if there is a way.